kahuna/public/js/components/gu-lazy-table/gu-lazy-table.js

```diff
@@ -201,9 +234,9 @@ lazyTable.controller('GuLazyTableCtrl', ['range', function(range) {
 }]);
 
 
-lazyTable.directive('guLazyTable', ['$window', 'observe$',
+lazyTable.directive('guLazyTable', ['$window', 'observe$', '$document',
```

Comment:
Not needed? Assuming it was used before the [element](https://github.com/guardian/grid/pull/1706/files#diff-e67d7a61b905b41b4f3d88c228e1ef95R293).